cis_v120/section_5.sp

```diff
@@ -11,7 +11,10 @@ benchmark "cis_v120_5" {
     control.cis_v120_5_1,
     control.cis_v120_5_2
   ]
-  tags = local.cis_v120_5_common_tags
+
+  tags = merge(local.cis_v120_5_common_tags, {
+    type = "Benchmark"
```

Comment:
Can we add the `service = "GCP/Storage"` tag here since all controls are for that service?